### PR TITLE
set-up and complete orders redux store

### PR DESCRIPTION
**Pull request by monicachoe**
order redux store 
approve after product/user redux store has been merged so we don't have conflicting directory structure

### CODE REVIEW

**Comment by monicachoe** [comment]
[Redux store reducer for orders](https://trello.com/c/dtWH4ssp/178-redux-store-reducer-for-orders)

File: client/store/reducers/order.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+		.catch(err => console.log(err));
+
+export const putOrder = order =>
+	dispatch =>
+		axios.put(`/api/orders/${order.id}`, order)
+		.then(res => dispatch(updateOrder(res.data || orders)))
+		.catch(err => console.log(err));
+
+// Reducer
+export default function (state = orders, action) {
```
Comment (itsdanni) [inline client/store/reducers/order.js:45]: Need action creator and thunk dispatcher for cancelling orders

File: client/store/reducers/order.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  switch (action.type) {
+    case GET_ORDER:
+      	return [...state, action.order];
+  	case GET_ORDERS:
+  		return action.orders;
+    case UPDATE_ORDER:
+    	return [...state, action.order];
```
Comment (itsdanni) [inline client/store/reducers/order.js]: For updated orders as well as deleted orders (to be implemented), the returned state should filter out the order by id (see products reducer)

**Review by itsdanni — CHANGES_REQUESTED** [review]
Need to deal with cancelling an order as well as edit the returned state from updating and cancelling orders

**Comment by itsdanni** [comment]
There are just some minor changes that need to be dealt with but github makes it look scary with the red x lol

File: client/store/reducers/order.js (hunk 1 above)
Comment (monicachoe) [reply client/store/reducers/order.js:45]: wouldn't that fall under the update order section because we're just changing the status in the model?

**Review by itsdanni — APPROVED** [review]
yay!

**Pull request merged**